clients/cli/src/main/java/org/apache/gravitino/cli/outputs/PlainFormat.java

```diff
@@ -231,4 +234,43 @@ public String getOutput(Audit audit) {
           audit.lastModifiedTime() == null ? "N/A" : audit.lastModifiedTime());
     }
   }
+
+  /**
+   * Formats an array of {@link org.apache.gravitino.rel.Column} into a six-column table display.
+   * Lists all column names, types, default values, auto-increment, nullable, and comments in a
+   * plain format.
+   */
+  static final class ColumnListPlainFormat extends PlainFormat<Column[]> {
+
+    /**
+     * Creates a new {@link ColumnListPlainFormat} with the specified output properties.
+     *
+     * @param context The command context.
+     */
+    public ColumnListPlainFormat(CommandContext context) {
+      super(context);
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public String getOutput(Column[] columns) {
+      String header =
+          COMMA_JOINER.join(
+              "name", "datatype", "default_value", "comment", "nullable", "auto_increment");
```

Comment:
"auto increment" only applies to an integer column,
how about print a field's data type as `integer (autoinc)`, rather than making `AutoIncrement` a column for all fields?

Reply:
I'd also go with empty values rather than "N/A" for columns with no default values. The suggestion regarding autoincrement is also a good one.